scripts/perftune.py

```diff
@@ -549,13 +563,27 @@ def _get_irqs(self):
         pass
 
 #### Private methods ############################
+    def __refresh_metadata_token(self):
+        self._metadata_token_time = datetime.datetime.now()
+        self._metadata_token = curl(self.EC2_METADATA_BASE_URL + "api/token", headers={"X-aws-ec2-metadata-token-ttl-seconds": self.EC2_METADATA_TOKEN_TTL}, method="PUT", timeout=0.1)
```

Comment:
Why these two are "protected" while the only place where you use them are in the `__instance_metadata(...)` below.

Unless I'm missing something they should be "private" together with the rest of the code.

Reply:
It was because of just copied the code from scylla-machine-image, fixed on the latest push